### PR TITLE
Bug/gsye 539 Handle case when user inputs 0 values for asset settings

### PR DESCRIPTION
**Pull request by hannesdiedrich**

### CODE REVIEW

File: gsy_framework/community_datasheet/row_converters.py [hunk 1]
```diff
@@ -217,9 +220,18 @@ def convert(cls, row: Dict) -> Dict:
         """
         cls._validate_row(row)
 
+        user_min_allowed_soc = row[StorageSheetHeader.MINIMUM_ALLOWED_SOC]
         min_allowed_soc = (
-            row[StorageSheetHeader.MINIMUM_ALLOWED_SOC]
-            or ConstSettings.StorageSettings.MIN_ALLOWED_SOC)
+            user_min_allowed_soc if user_min_allowed_soc not in NULL_VALUES
```
Comment (mrsaemir) [inline gsy_framework/community_datasheet/row_converters.py:225]: I'm tempted to suggest to convert these two lines to a separate method and use it everywhere else. What do you think?

**Review by mrsaemir — COMMENTED** [review]
LGTM

File: gsy_framework/community_datasheet/row_converters.py (hunk 1 above)
Comment (hannesdiedrich) [reply gsy_framework/community_datasheet/row_converters.py]: Done [9adcde3](https://github.com/gridsingularity/gsy-framework/pull/429/commits/9adcde361121cbefe62f11796199558dac3edb85)

**Review by mrsaemir — APPROVED** [review]
LGTM

**Pull request merged**